app/controllers/follower_accounts_controller.rb

```diff
@@ -17,9 +19,9 @@ def index
       end
 
       format.json do
-        raise Mastodon::NotPermittedError if params[:page].present? && @account.user_hides_network?
+        raise Mastodon::NotPermittedError if page_requested? && @account.user_hides_network?
 
-        expires_in 3.minutes, public: true if params[:page].blank?
+        expires_in page_requested? ? 0 : 3.minutes, public: !authorized_fetch_mode?
```

Comment:
what's the point of changing the expires_in when a page is requested?

Reply:
When no page is required, what we return is essentially a counter--and these counters are requested when we're fetching actor info (so that endpoint can be hammered quite a bit). When a page is requested, it returns actual data, and I guess we don't want actual data to be outdated.